app/build.gradle

```diff
@@ -32,6 +32,7 @@ android {
 }
 
 dependencies {
+    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
 
```

Comment:
This empty line can be removed to keep all the "implementation" libraries grouped together. You can add newlines where the type of library changes. E.g: between "implementation" and "testImplementation"